backend/clubs/views.py

```diff
@@ -4802,25 +4827,14 @@ def cart(self, request, *args, **kwargs):
                                     allOf:
                                         - $ref: "#/components/schemas/Ticket"
                                 sold_out:
-                                    type: list
+                                    type: array
                                     items:
                                         type: object
                                         properties:
-                                            id:
-                                                type: integer
                                             event:
-                                                type: object
-                                                properties:
-                                                    id:
-                                                        type: integer
-                                                    name:
-                                                        type: string
+                                                type: integer
                                             type:
                                                 type: string
-                                            owner:
-                                                type: string
-                                            price:
-                                                type: number
                                             count:
                                                 type: integer
```

Comment:
@joyliu-q looks like you're dealing with the annotated queryset (see L4865), so these fields won't exist. Unsure if this is the behavior you intended.

Reply:
Fixing this shortly

Reply:
Thanks! Looks like there were some merge conflicts as a result of your changes. Resolved conflicts in [a9054ea](https://github.com/pennlabs/penn-clubs/pull/675/commits/a9054eab62c8e7c2556059ba5b9796fe7bf68d9a) and modified tests in [63ff452](https://github.com/pennlabs/penn-clubs/pull/675/commits/63ff45235b498aaa77026da50e7dd3fa0eb99b04).